src/s3b_the_roach_market.tw2

```diff
@@ -344,3 +357,46 @@ I’d found this same box on this same table as a kid, and snatched it up immedi
 Honestly I was kinda hurt to see the box still on the table, but... whatever. Today’s ninjas had no fuckin’ taste.
 
 [[(back)|RM-junkheap-goodshit]]
+
+(set: $junkHeapItemsSeen to it +1)
+
+::RM-junkheap-flashback
+I walked up to the scooter and just took it in for a bit. It wasn't in good shape, nothing in this pile could say that.
+
+It was beaten to hell, with rust gathering all over it. The light had been busted out long ago by some ninja looking for a quick thrill. I felt myself pulled back to the [first time I saw the scooter]<scooterTrigger|.
+
+<div class="flashback">(click: ?scooterTrigger)[$dissolve[It’s a couple years ago and I’m waiting outside of the school trailer. I’m waiting for Anna, as usual.
+
+We’re gonna head out to the far fields – where you can actually see the river. The only lights are a Fed boat doing a routine patrol and the only people dudes who’ve wandered too far on a bad trip.
```

Comment:
I like the dash for emphasis, but the second phrase doesn't really require it. What if we added to it:

"We’re gonna head out to the far fields – to where you can actually see the river."

Reply:
Also, for clarity: "The only lights we could see came from a Fed boat running a routine patrol in the distance, and the only people around were a few dudes who’d wandered too far on a bad trip."

Reply:
Fixed